service/service.go

```diff
@@ -92,6 +101,12 @@ func (svc *Service) Close(ctx context.Context) error {
 		}
 
 		// ADD CODE HERE: Close other dependencies, in the expected order
+		if svc.ServiceList.MongoDB {
+			if err := svc.MongoDB.Close(ctx); err != nil {
+				log.Event(ctx, "error closing mongoDB", log.Error(err), log.ERROR)
```

Comment:
to be consistent with the log in `registerCheckers` below
```suggestion
				log.Event(ctx, "error closing mongo db", log.Error(err), log.ERROR)
```